pkg/webhooks/webhookpatch.go

```diff
@@ -113,8 +123,8 @@ func (w *WebhookCertPatcher) updateWebhookHandler(oldConfig, newConfig *v1.Mutat
 }
 
 func (w *WebhookCertPatcher) addWebhookHandler(config *v1.MutatingWebhookConfiguration) {
-	for i, wh := range config.Webhooks {
-		if strings.HasSuffix(wh.Name, w.webhookName) && !bytes.Equal(config.Webhooks[i].ClientConfig.CABundle, w.caCertPem) {
+	for _, wh := range config.Webhooks {
+		if strings.HasSuffix(wh.Name, w.webhookName) {
```

Comment:
Copying my comment which somehow got hidden:

I still don't think its useless.. its good to skip updates when they are not needed, updates are fairly expensive.

Even if we are not triggering updates, they can happen for all sorts of reasons. Why not add a simple 1 line check to avoid a full api server call?